tools/ldb_test.py

```diff
@@ -340,6 +366,24 @@ def testDumpLoad(self):
         self.assertFalse(self.dumpDb(
             "--db=%s --create_if_missing" % origDbPath, dumpFilePath))
 
+        # Dump and load with BlobDB enabled
+        blobParams = " ".join(["--enable_blob_files", "--min_blob_size=1",
+                                "--blob_file_size=2097152"])
+        dumpFilePath = os.path.join(self.TMP_DIR, "dump9")
+        loadedDbPath = os.path.join(self.TMP_DIR, "loaded_from_dump9")
+        self.assertTrue(self.dumpDb(
+            "--db=%s %s" % (origDbPath, blobParams), dumpFilePath))
+        self.assertTrue(self.loadDb(
+            "--db=%s %s --create_if_missing" % (loadedDbPath, blobParams),
+            dumpFilePath))
+        self.assertTrue(self.loadDb(
+            "--db=%s %s" % (loadedDbPath, blobParams),
+            dumpFilePath))
```

Comment:
I think this second `load` is redundant

Reply:
In my test, the `.blob` file won't generate if load it just once. (The data may be cached in the WAL) This is a trick to force generating the `.blob` file. I learned this from the `testSSTDump` method

Reply:
I believe loading with `disable_wal` set would solve that issue.